src/olympia/devhub/tests/test_tasks.py

```diff
@@ -567,7 +567,7 @@ def test_run_linter_use_temporary_file_large_output(self):
 
             assert tmpf.call_count == 2
             assert result['success']
-            assert result['warnings'] > 700
+            assert result['warnings'] == 11
```

Comment:
But is the test still relevant then ? The comment above says that the add-on generates a lot of warnings, and it's testing that a `TemporaryFile` is used in this scenario. So at least the comments needs to be updated because it's no longer true, but also, why would a temporary file be used if the number of warnings is significantly lower ?